### PR TITLE
Weighted moving average smoothing of spectra

### PR DESCRIPTION
**Pull request by SiggiSmara**
As explained in #272

### CODE REVIEW

File: R/functions-Spectrum.R [hunk 1]
```diff
@@ -439,7 +439,7 @@ pickPeaks_Spectrum <- function(object, halfWindowSize = 2L,
 }
 
 smooth_Spectrum <- function(object,
-                            method = c("SavitzkyGolay", "MovingAverage"),
+                            method = c("SavitzkyGolay", "MovingAverage","MovingWeightedAverage"),
```
Comment (sgibb) [inline R/functions-Spectrum.R]: Please add a space after the comma. We try to follow the Bioconductor coding style: https://bioconductor.org/developers/how-to/coding-style/

File: man/smooth-methods.Rd [hunk 2]
```diff
@@ -26,7 +27,8 @@
 
       Smooths all spectra in \code{MSnExp}. \code{method} could be
       \code{"SavitzkyGolay"} or
```
Comment (sgibb) [inline man/smooth-methods.Rd]: Please remove the "or" and replace it by a comma, reads: SavitzkyGolay, MovingAverage or MovingWeightedAverage.

**Comment by SiggiSmara** [comment]
Thanks for the helpful comments. I'm working on an updated pull also reflecting the suggestions in sgibb/MALDIquant#54.

**Comment by sgibb** [comment]
I guess this PR is not needed anymore. MALDIquant 1.17 introduces a new "weighted" argument to `MALDIquant:::.movingAverage` (which would be passed by `...`). I close this. @SiggiSmara if I am wrong please feel free to reopen.

**Comment by SiggiSmara** [comment]
The only thing that was to do in the end was to change the man page to reflect the changes in MALDIquant.  I leave it up to you guys if you want to reopen this one or change the man pages at a later time, perhaps when something more substantial is to be done there?

**Pull request closed without merging**